.travis.yml

```diff
@@ -7,7 +7,6 @@ cache: cargo
 sudo: false
 env:
   global:
-    - RUSTFLAGS="--deny warnings"
```

Comment:
I've removed the since it was causing issues on nightly. Interestingly, some of the new lints generated by `svd2rust` have been removed in nightly and show up as warnings. 
Alternatively, we could allow failures on nightly. I'm ok with either solution atm.